docs/node-operation/itn-manual.md

```diff
@@ -0,0 +1,210 @@
+
+### Monthly rewards pot
+
+For each month of the ITN, a specific amount of FACT tokens will be allocated
+for validator rewards. The first month will make 1,500,000 $FACT available for
+ITN participants to claim. This amount will be adjusted monthly to balance the
+rewards, based on the required effort and actual validator participation.
+
+### Rewards calculation method
+
+Rewards will be calculated based on the daily number of collections,
+validations, and publications, multiplied by each validator’s ITN
+[multiplier](https://status.orcfax.io/validators).
+
+The actual rewards accrued per validator will be proportional to their
+participation in a given day relative to all other validators.
+
+```text
+Example
+if a validator has collected 5% of all fact statements on a specific day,
+they will receive 5% of that day’s available rewards (after applying the ITN
+multiplier). This method ensures that optimal uptime and efficient validator
+setup will yield the highest possible rewards.
+```
+
+Validators will be able to use the Orcfax Validator Dashboard to track their
+metrics and corresponding rewards. You can find a basic ITN rewards calculator
+[here][calc-1].
+
+[calc-1]: https://docs.google.com/spreadsheets/d/1aH4Zwtn8KUTtrdzBBZFK1_Kulb7_a4uJGjzLhmPTLFc/edit?source=post_page-----9bc1506da743--------------------------------&gid=1978771855#gid=1978771855
+
+### Maintenance periods
+
+During announced maintenance periods, such as when Orcfax transitions to a new
+ITN phase or addresses specific issues, no rewards will be distributed. Excess
+rewards not distributed for a given month due to these situations will be moved
+back to the validator allocation.
+
+If validators are unable to earn rewards due to unscheduled maintenance or
+unexpected issues, they will receive equal rewards for those days.
+
+```text
+Example
+If on a given day there are 60 registered ITN participants, and the network
+experiences unscheduled maintenance or unexpected issues, the available rewards
+for that day will be equally divided among participants.
+```
+
+### Reward claiming process
+
+As participants accumulate rewards, they will be able to track them on our
+Validator Dashboard, and will be able to claim their accrued rewards at any
+given time.
+
+This approach prevents scheduled monthly releases that might affect price action
+and cause more volatility.
+
+To claim rewards:
+
+1. Send small amount of ADA to a given address (covering the minimum UTXO and
+transaction fees)
+1. Shortly after, participants will receive their $FACT rewards.
+
+This process can be repeated as often as a validator wishes to claim their
+rewards.
+
+#### Vesting method
+
+We are investigating whether the addition of a vesting method to the claiming
+process will be possible before ITN goes live. If we find a way to add this
+feature, we will implement it. However, we cannot guarantee at this moment that
+we will be able to do so.
```

Comment:
if we're not sure if we can do this, do we need to mention it at all? (or another way, who is asking for this?)

Reply:
Good point! I had pulled this out of a medium article but was unfamiliar with the context-- I think removing altogether is best.